src/crypto/tls/testdata/Client-TLSv10-ClientCert-ECDSA-ECDSA

```diff
@@ -17,11 +17,11 @@
 000000f0  07 30 ff f6 84 af c4 cf  c2 ed 90 99 5f 58 cb 3b  |.0.........._X.;|
 00000100  74                                                |t|
 >>> Flow 2 (server to client)
-00000000  16 03 01 00 59 02 00 00  55 03 01 09 68 1e 95 bd  |....Y...U...h...|
-00000010  f3 a3 32 97 d0 26 e7 c7  a5 5c b2 31 6c e2 07 41  |..2..&...\.1l..A|
-00000020  95 59 75 18 52 a8 7d 77  5a 1e be 20 35 f2 d1 f7  |.Yu.R.}wZ.. 5...|
-00000030  cf 70 2a 47 5a 5c bb 36  51 4c 5e f4 82 f4 db 12  |.p*GZ\.6QL^.....|
-00000040  85 7f d6 14 64 f2 e0 68  cd d0 60 1a c0 09 00 00  |....d..h..`.....|
+00000000  16 03 01 00 59 02 00 00  55 03 01 27 14 34 e9 ef  |....Y...U..'.4..|
+00000010  89 88 53 1c 48 31 03 98  8c 7c 56 63 bc 6b 2c 86  |..S.H1...|Vc.k,.|
+00000020  63 39 d2 0f 1d a0 b8 6b  c5 11 84 20 43 83 47 5b  |c9.....k... C.G[|
+00000030  ab bb 57 2a 30 7d 37 3f  a1 44 56 87 fb 14 33 8d  |..W*0}7?.DV...3.|
+00000040  72 cb 79 61 ee 4b fd 83  66 e8 e7 a2 c0 09 00 00  |r.ya.K..f.......|
```

Comment:
Yes, it is needed. All the tests fails if they are no re-generated. I'm not sure why is it needed to review the test vectors instead of just ignoring this files.. so is there a reason why you are reviewing them?

Reply:
Not necessary, just trying to find ways to reduce the scope. Thanks for clarifying, you can disregard this comment.